pkgs/development/tools/oq/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, crystal, libxml2 }:
+
+crystal.buildCrystalPackage rec {
+  pname = "oq";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "Blacksmoke16";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0wc1wdvqybgybq09dr78b93mdyvr7avya5w8kffbv8p4i9yyrfn4";
+  };
+
+  buildInputs = [ libxml2 ];
```

Comment:
Consider adding `jq` to buildInputs or propagatedBuildInputs 

(I'm not too sure about how $PATH is handled in crystal packaging)

Reply:
It fails on environments that don't have `jq` in the $PATH. (Mine did when testing)

```
$ ./results/oq/bin/oq 
oq error: execvp (jq): No such file or directory: No such file or directory
```

Reply:
Sure, didn't notice that. Thanks!

Reply:
The convo is marked as resolved, but can't see any fix for this problem.